### DISCUSSION TITLE
How do I disable documentation popups?

### DISCUSSION THREAD

**Discussion by KenHV** [neoclide/coc.nvim#2778] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52503501/103396361-d0f73800-4b58-11eb-9954-8ce8523a5659.png)
This popup gets in the way a lot, I searched past issues and the suggested solution is `set completeopt-=preview` but that doesn't seem to do anything.

**Comment by fannheyward** [comment, marked as answer]
It's signatureHelp, set `"signature.target": "echo"` in your coc-settings.json.

**Reply by KenHV** [reply]
Thank you!
